gammapy/image/models/tests/test_models.py

```diff
@@ -74,11 +87,22 @@ def test_skyshell2D():
     desired = 0.0002976757280439522 / u.deg ** 2
     assert_quantity_allclose(actual, desired)
 
+    lons = np.arange(-5, 5) * u.deg
+    lats = np.arange(0, 5) * u.deg
+    lon, lat = np.meshgrid(lons, lats)
+    actual = model(lon, lat)
+    assert actual.shape == (5, 10)
+
 
 @requires_dependency('scipy')
 @requires_data('gammapy-extra')
 def test_template2d():
     filename = ('$GAMMAPY_EXTRA/datasets/catalogs/fermi/Extended_archive_v18'
                 '/Templates/HESSJ1841-055.fits')
     template = SkyTemplate2D.read(filename)
-    assert_allclose(template(26.7, 0), 1.1553735159851262)
+    lon = 26.7 * u.deg
+    lat = 0 * u.deg
+    actual = template(lon, lat)[0]
+    desired = 0.00017506744188722223 / u.deg ** 2
+    #desired = 1.1553735159851262 / u.deg ** 2
```

Comment:
@cdeil The test value changed quite a bit here. You wrote the previous version of the TemplateModel class. Any idea why this is?